controllers/slicegateway/slicegateway.go

```diff
@@ -534,7 +564,7 @@ func (r *SliceGwReconciler) deploymentForGatewayClient(g *kubeslicev1beta1.Slice
 	return dep
 }
 
-func (r *SliceGwReconciler) GetGwPodNameAndIP(ctx context.Context, sliceGw *kubeslicev1beta1.SliceGateway) (string, string) {
+func (r *SliceGwReconciler) GetGwPodNameAndIP(ctx context.Context, sliceGw *kubeslicev1beta1.SliceGateway) ([]string, []string) {
```

Comment:
func name could be more accurate?
GetGwPodNamesAndIPs?

Reply:
Will change it